### DISCUSSION TITLE
Could you make a guide on how to use the tool?

### DISCUSSION THREAD

**Discussion by sylvesterroos** [DATL4G/DXVKoTool#7] [Question] [answered]
For example, my Steam games are not showing up. I thought that adding the root of the game folder would cause it to show up, but nothing happened after I added it.

P.S
Is this the correct place to put this question? I'm not sure what you mean with the 'Projects' category

**Comment by DatL4g** [comment, marked as answer]
Yes discussion is fine for that, an issue would be okay as well.
GitHub recently removed the "real" Projects feature, so I keep track of adding new features like [this](https://github.com/DATL4G/DXVKoTool/discussions/5).

Steam games are different compared to "normal" games.
DXVK caches from steam games are stored in a separate folder, so I have to read a config file for each game to find it's caches.

You can add the shader cache folder and connect it manually with a repo item.
Shader caches are stored under ```~/.steam/steam/steamapps/shadercache``` or ```~/.var/com.valvesoftware.Steam/.steam/steam/steamapps/shadercache```

**Reply by sylvesterroos** [reply]
I've tried a couple of different things but I can't get it to work. These seem to be the logical steps to me:
1. Click the + which opens the 'select game folder' picker
2. Select the shader cache folder per your comment (1172470 is Apex Legends)
![image](https://user-images.githubusercontent.com/39904003/193637993-062068cd-0ec7-4bd9-9c44-45a120ab3946.png)
3. I expect the game to show up, but nothing happens

Where am I going wrong?

**Reply by DatL4g** [reply]
You did it correctly, but there was a bug with manually added games.

Fixed in the latest release [v0.3.1-beta02](https://github.com/DATL4G/DXVKoTool/releases/tag/v0.3.1-beta02)
